src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy

```diff
@@ -158,12 +158,12 @@ class MirrorEndpointsSpec extends HealthCheckSpecification {
         and: "Original flow rule counter is not increased"
         flowRule.packetCount == findFlowRule(getFlowRules(swPair.src.dpId), mirrorDirection).packetCount
 
-        //https://github.com/telstra/open-kilda/issues/4517
-//        and: "System collects stat for mirror cookie in otsdb"
-//        Wrappers.wait(statsRouterRequestInterval) {
-//            def tags = [flowid: flow.flowId, cookie: mirrorRule.cookie]
-//            assert otsdb.query(genTrafficTime, metricPrefix + "flow.raw.bytes", tags).dps.size() > 0
-//        }
+        // https://github.com/telstra/open-kilda/issues/4517
```

Comment:
Why is this comment here?

Reply:
This comment was there before I started an issue. I thought it is important for QA

Reply:
It explained why certain part of the test was commented and also marked part of the test that can verify certain issue. As soon as issue is fixed the comment can be removed

Reply:
Thanks for the explanation